cypress/e2e/home_page.cy.ts

```diff
@@ -134,4 +128,14 @@ describe("The Home Page", () => {
       .next()
       .should("contain.text", 88);
   });
+
+  it("displays a dynamic heading", () => {
+    cy.get("[data-cy='Action']").click();
+    cy.contains("Platforms").click();
```

Comment:
Consider switching to `data-` selectors to be consistent and follow Cypress's recommendations for best practices.